jena-text/src/test/java/org/apache/jena/query/text/assembler/TestGenericAnalyzerAssembler.java

```diff
@@ -0,0 +1,198 @@
+package org.apache.jena.query.text.assembler;
```

Comment:
I've made these changes. I only added the `JenaSystem.init()` to `TestGenericAnalyzerAssembler.java` but am happy to add to the other tests in jena-text.